lib/test.js

```diff
@@ -209,8 +209,14 @@ Test.prototype._assert = function assert (ok, opts) {
         var e = new Error('exception');
         var err = (e.stack || '').split('\n');
         var dir = path.dirname(__dirname) + '/';
```

Comment:
specifically here - instead of using a hardcoded slash, we could use `path.sep`?